### ISSUE TITLE
Mix insertRow and editable plugin

### ISSUE THREAD

**Issue by albfan** [wenzhixin/bootstrap-table#6224]
### Description

x-editable has an specific section for new records:

https://vitalets.github.io/x-editable/docs.html#newrecord

All look relevant, (deal with no primary key, submit manually)

I tried a working example with bootstrap-table:

There's no concept of create new record on bootstrap table editable.

I workaround the problem keeping pk returned on first column edition, and filling it with a function params, but sound interesting thing about general solution for this use case.

Basically if pk is undefined:
- store column data even without submit
- allow to get pk from some parent structure for row
- row column validation

I can create a basic working example if needed

**Comment by UtechtDustin** [comment]
As x-editable is no more in support/is eol, i don't think it makes sense to spend time to add new features for it.
In the Further we want to replace it with a lib which will be actively developed or we will develop it by ourselves.

I'm sorry we can't help here.
